tools/check-config-options.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+srcdir=${1:-.}
+
+get_supported_options()
+{
+	awk '/START OPTION LIST/{flag=1;next}/END OPTION LIST/{flag=0}flag' "$srcdir/src/nm-config.c" | \
```

Comment:
You don't need the trailing slash -- the shell will proceed on the next line when it expects a command after a `|`